deploy/helm/edot-collector/kube-stack/values.yaml

```diff
@@ -568,7 +587,8 @@ collectors:
         elasticsearch/otel:
           endpoints: # List of Elasticsearch endpoints.
             - ${env:ELASTIC_ENDPOINT}
-          api_key: ${env:ELASTIC_API_KEY} # API key for Elasticsearch authentication.
+          auth:
+            authenticator: bearertokenauth
```

Comment:
I'm no longer super familiar with the ES exporter but glancing through the code, I'm wondering if this actually works as expected.  

The reason I ask that is I see `config.Authentication.APIKey` being used when creating the Elasticsearch client but I'm not seeing where the API key from the authenticator is being provided to the Elasticsearch client.